bsp/boards/common/aes_ccms.c

```diff
@@ -0,0 +1,55 @@
+/**
+  \brief AES CCMS implementation
+  
+  \author Marcelo Barros de Almeida <marcelobarrosalmeida@gmail.com>, March 2015.
+ */
+#include <string.h>
+#include <stdint.h>
+#include "crypto_engine.h"
+
+int aes_ccms_enc(uint8_t* a,
```

Comment:
please provide Doxygen-style documentation of the functions (applies throughout)